src/Nervos/SignatureVersion.h

```diff
@@ -0,0 +1,11 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+namespace TW::Nervos {
+enum SignatureVersion { BASE, WITNESS_V0 };
```

Comment:
hmm, does Nervos have different signature type like bitcoin? this seems not used?

Reply:
Nope. Removed it.